common/tier4_planning_rviz_plugin/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@ find_package(Qt5 REQUIRED Core Widgets)
 set(QT_LIBRARIES Qt5::Widgets)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-add_definitions(-DQT_NO_KEYWORDS)
 find_package(Eigen3 REQUIRED)
 
 include_directories(
```

Comment:
The removal of this line is intended and needed as this definition leads to the usage of `QSLOT` instead of `slots` and the file `<rviz_rendering/render_window.hpp>` is using slot. With using this definition, the new plugin is not compiling.